addons/website_slides/controllers/main.py

```diff
@@ -963,17 +963,32 @@ def slide_category_search_read(self, fields, domain):
             'can_create': can_create,
```

Comment:
Argh I was hoping to get out of that one ! I don't really know why unfortunately ... I only know that they are fucked when widgets are attached to newly added dom elements and jsonRpc saved my butt :( . 

A solution to remove that would be to let the course_content_manager handle the rpc calls which would not be such a bad idea imo.

Let me know what you think.

Reply:
is it because this.slideId is unknown ? or because this.rpc is not (well?) defined ?
Letting the manager handle the rpc calls is maybe not a bad idea... but this will lead to a complete refactoring for all course_content_manager's children